packages/protobuf-bench/README.md

```diff
@@ -1,14 +0,0 @@
-# Code size comparison
```

Comment:
Did you mean to delete this file?

Reply:
It was gone after a `make setversion SET_VERSION=1.3.2` and is now back after a `make`. I'm going to blame a bad cache.